### PR TITLE
Document change that Contact.data.id is the ID used when creating Payments or Payment Requests

### PR DESCRIPTION
**Pull request by trev**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -5417,7 +5419,8 @@ func main() {
     {
       "ref": "PR.4",
       "initiator_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
-      "authoriser_id": "fb6a9252-3818-44dc-b5aa-2195391a746f",
+      "authoriser_id": "de86472c-c027-4735-a6a7-234366a27fc7",
+      "authoriser_contact_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
```
Comment (antulik) [inline source/index.html.md]: is it normal that initiator_id and authoriser_contact_id are identical?

File: source/index.html.md [hunk 2]
```diff
@@ -5431,7 +5434,8 @@ func main() {
     {
       "ref": "PR.5",
       "initiator_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
-      "authoriser_id": "fb6a9252-3818-44dc-b5aa-2195391a746f",
+      "authoriser_id": "de86472c-c027-4735-a6a7-234366a27fc7",
+      "authoriser_contact_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
```
Comment (antulik) [inline source/index.html.md]: ditto

File: source/index.html.md [hunk 3]
```diff
@@ -5923,10 +5927,15 @@ func main() {
...
 |»»» description|body|string|true|Description that both the payer an recipient can see|
-|»»» recipient_id|body|string|true|Contact bank account to pay (`Contact.data.bank_account.id`)|
+|»»» recipient_contact_id|body|string|true|Contact to pay (`Contact.data.id`)|
 |»»» metadata|body|[Metadata](#schemametadata)|false|Use for your custom data an certain Split customisations|
 |»» metadata|body|[Metadata](#schemametadata)|false|Use for your custom data an certain Split customisations|
 
+#### Detailed descriptions
+
+**»» Payout**: The actual Payout
+<aside class="notice">[23/07/2018] Please note that we've deprecated <code>recipient_id</code> in favour of <code>recipient_contact_id</code>. We will continue to support the use of <code>recipient_id</code> but encourage you to use <code>recipient_contact_id</code> to identify the recipient of the payout.</aside>
```
Comment (antulik) [inline source/index.html.md]: is it worth mentioning that they accept different data?

File: source/index.html.md (hunk 1 above)
Comment (trev) [reply source/index.html.md]: Good 👁

**Pull request merged**